### PR TITLE
add corrected applicator to linear alignment

### PR DESCRIPTION
**Pull request by christophmluscher**
@Marvin84 can I apply this like this to the linear alignment step?

### CODE REVIEW

**Comment by Marvin84** [comment]
> @Marvin84 can I apply this like this to the linear alignment step?

Looks good.
Once you ran this, please send me your `alignment.config` of the resulting job so that I can double-check.

File: am/config.py [hunk 1]
```diff
@@ -113,7 +113,8 @@ def get_align_config_and_crp_for_corrected_applicator(
     """
     Set the correct type of applicator, default is "legacy". Moreover, set exit penalities to zero
     For a given word sequence the exit penalty is constant with respect to the max/sum
-    :param config:
+    :param crp:
+    :param exit_penalty:
```
Comment (Atticus1806) [inline am/config.py:117]: Could you add some docs pls? :)

**Comment by Atticus1806** [comment]
I have to be honest. I have no clue :sweat_smile:

File: am/config.py (hunk 1 above)
Comment (Marvin84) [reply am/config.py:117]: Docs of what type you mean?

**Comment by Marvin84** [comment]
> I have to be honest. I have no clue 😅

There is a `rasr` PR for this : https://github.com/rwth-i6/rasr/pull/14

**Comment by Atticus1806** [comment]
> There is a rasr PR for this : https://github.com/rwth-i6/rasr/pull/14

Thanks, but I was referring to the python test failing or rather why this only comes up now ^^

**Review by Atticus1806 — CHANGES_REQUESTED** [review]
So if we bump to 3.8 I would revert the "fix" to make the test run.

**Review by Atticus1806 — APPROVED** [review]

**Pull request merged**